docs/component-stability.md

```diff
@@ -260,6 +260,44 @@ To move within the 'Maintained' ladder ("graduate"), the process for doing so is
 3. If approved, a PR to change the stability level should be opened and MUST be approved by all
    listed code owners.
 
+## Graduation criteria
+
+In addition to the requirements outlined above, additional criteria should be met before a component
+can graduate to a higher stability level. These ensure that the component is ready for the increased
+usage and scrutiny that comes with a higher stability level, and that the community around it is
+sufficiently healthy.
+
+If the graduation criteria are not met, the approver should provide feedback on what is missing and
+how to address it. The component owners can then address the feedback and re-request graduation on
+the same issue.
+
+## In development to alpha
+
+No additional criteria are required to graduate from development to alpha.
+The component still needs to meet the general requirements for alpha components.
+
+## Alpha to beta
+
+To graduate any signal from alpha to beta on a component:
+1. The component MUST have at least two active code owners.
+2. The code owners for non-vendor-specific components SHOULD have at least two different employers.
```

Comment:
I believe we previously removed all language pertaining to the notion of "vendor-specific" components. Is this a meaningful distinction at this point?

Reply:
Would your suggestion be to remove this requirement entirely? Or to make it apply to all components? (The latter feels difficult to me)

Reply:
I think it could be removed entirely but this isn't something I feel strongly about. My main point is that it is an ambiguous requirement since we don't have a definition for vendor-specific component.

Reply:
Removed, we can always add this later if we find a good definition of vendor-specific